Gaijinizer patch/Map536.txt

```diff
@@ -4,1053 +4,1342 @@
 
 本物の白兎は、\\C[10]もう死んでる。
-> CONTEXT: Map536/events/7/pages/0/680/Dialogue/enemy29 < UNTRANSLATED
-
+> CONTEXT: Map536/events/7/pages/0/680/Dialogue/enemy29
+And it looks like even you've failed to realize
+something. The real White Rabbit \\C[10]is already dead\\C[0].
 > END STRING
 
 > BEGIN STRING
 あの世界は、とうとう\\C[10]最後の一巡\\C[0]に入ったよ。
 
 そのチャンスを残すため、白兎は力を使い果たして死んだ。
-> CONTEXT: Map536/events/7/pages/0/687/Dialogue/enemy29 < UNTRANSLATED
+> CONTEXT: Map536/events/7/pages/0/687/Dialogue/enemy29
+That world has finally gone into its \\C[10]final round\\C[0].
 
+White Rabbit used up all of her power and died
+in order to make that chance happen.#RonIffy
 > END STRING
 
 > BEGIN STRING
 彼女があそこまでルカ君達に入れ込むなんて、意外だったね。
 まぁ、今のボクも人の事言えないけど……
 
 『白兎』と接触したキミなら、それも感じ取ったのかな……？
-> CONTEXT: Map536/events/7/pages/0/691/Dialogue/enemy29 < UNTRANSLATED
-
+> CONTEXT: Map536/events/7/pages/0/691/Dialogue/enemy29
+It's quite unlike her to invest so much of herself
+into Luka-kun and his friends. But then I'm not
+one to talk. You must have sensed it as well
+since you've had contact with her.#Ron
 > END STRING
 
 > BEGIN STRING
 ただ……キミが白兎じゃないって分かっていても……
 
 ボクは、\\C[2]ルカくんの協力が欲しかった。
 だからキミに話を合わせて、お茶会の手はずを整えた……
-> CONTEXT: Map536/events/7/pages/0/696/Dialogue/enemy29 < UNTRANSLATED
-
+> CONTEXT: Map536/events/7/pages/0/696/Dialogue/enemy29
+But even though I knew you weren't really White
+Rabbit, \\C[2]I still wanted to help Luka-kun. That's
+why I went along with your story and arranged
+the Tea Party.#Ron
 > END STRING
 
 > BEGIN STRING
 それにしても、
 『異世界間で魔力をコンバートする』か………
 
 キミも器用な事するよね……
-> CONTEXT: Map536/events/7/pages/0/701/Dialogue/enemy29 < UNTRANSLATED
-
+> CONTEXT: Map536/events/7/pages/0/701/Dialogue/enemy29
+But still... to convert magic in the space of
+another world... like her, you can do some fairly
+clever things.#Ron
 > END STRING
 
 > BEGIN STRING
 ………全部バレてた、か……
 
 なーんだ、それじゃあこの姿で来る必要なかったかな……
-> CONTEXT: Map536/events/7/pages/0/707/Dialogue/sirousagi_fc1 < UNTRANSLATED
+> CONTEXT: Map536/events/7/pages/0/707/Dialogue/sirousagi_fc1
+Looks like the cat's out of the bag...
 
+Oh man, that means there wasn't any point in
+taking this form...#Ron
 > END STRING
 
 > BEGIN STRING
 でも、結果的に良かったでしょ？
 
 \\C[2]アバドーンですら、ボクを白兎だと誤認したし、
 ボクの存在が薄れても、３人のつながりは弱まらなかった。
-> CONTEXT: Map536/events/7/pages/0/711/Dialogue/sirousagi_fc1 < UNTRANSLATED
-
+> CONTEXT: Map536/events/7/pages/0/711/Dialogue/sirousagi_fc1
+But it all worked out in the end.
+\\C[2]Even Abaddon mistook me for White Rabbit,
+and when my existence was fading, those three
+kept their connections to each other.#RonIffy
 > END STRING
 
 > BEGIN STRING
 \\C[2]だってボクは、もともとマイルームに居たんだもの
 
 魔力コンバート以外は、『つながり』に何の影響もないのさ。
-> CONTEXT: Map536/events/7/pages/0/716/Dialogue/sirousagi_fc1 < UNTRANSLATED
+> CONTEXT: Map536/events/7/pages/0/716/Dialogue/sirousagi_fc1
+That's because I was always in My Room.
 
+So other than my magic conversion, my presence
+or absence didn't affect the "connection".#RonIffy
 > END STRING
```

Comment:
Minor changes, and adding back in the color tags appropriately:
```
That's because \\C[2]I've always been in My Room.\\C[0]

So other than my magic conversion, my presence
or absence didn't affect the "connection".

```